packages/sgugpushdpopd/SPECS/sgugpushdpopd.spec

```diff
@@ -0,0 +1,37 @@
+Name:           sgugpushdpopd
+Version:        0.1
+Release:        1%{?dist}
+Summary:        Adds pushd and popd to sgug-rse
+
+License:        GPL
+Source0:        pushd 
+Source1:        popd
+
+BuildArch:      noarch
+
+
+
+%description
+
+
+%prep
+cp -av %{SOURCE0} .
+cp -av %{SOURCE1} .
+
+
+#%build
+
+
+%install
+install -m 755 -p -D %{SOURCE0} $RPM_BUILD_ROOT/usr/sgug/bin/pushd
```

Comment:
there is totally a better macro for this path but I don't know that I care (the sgug bit)